x-pack/plugins/security_solution/public/common/components/links_to_docs/links_translations.ts

```diff
@@ -41,3 +41,11 @@ export const ML_JOB_COMPATIBILITY_LINK_TEXT = i18n.translate(
     defaultMessage: 'ML job compatibility',
   }
 );
+
+export const COVERAGE_OVERVIEW_LINK_PATH = 'rules-coverage.html';
+export const COVERAGE_OVERVIEW_LINK_TEXT = i18n.translate(
+  'xpack.securitySolution.documentationLinks.coverageOverview.text',
+  {
+    defaultMessage: 'Learn more.',
```

Comment:
Nit: Oh interesting, this shows that link text sometimes can be very context-specific, and ideally should be passed as a property to the `CoverageOverviewLink` and other link components. Just something we could refactor in the future.